### PR TITLE
2021.12 release notes: Add breaking changes in the statistics component

### PR DESCRIPTION
**Pull request by ThomDietrich**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by ThomDietrich** [comment]
@frenck in case you didn't see this automatically

File: source/_posts/2021-12-11-release-202112.markdown [hunk 1]
```diff
@@ -267,6 +267,10 @@ noteworthy changes this release:
   - Selects can now have a single or no options ([@emontnemery])
   - Added 'trigger' support for Alarm Control Panels ([@XaF])
   - Added `lock.open` support ([@Hypnos3])
+- The statistics component received a bunch of new statistical characteristics
+  and one can be chosen to be presented as the state of the sensor directly.
+  For performance reasons the full set of characteristcs was removed from
+  attributes ([@ThomDietrich]).
```
Comment (frenck) [inline source/_posts/2021-12-11-release-202112.markdown]: I was not picked to be added here. Lets remove this part.

```suggestion
```

File: source/_posts/2021-12-11-release-202112.markdown [hunk 2]
```diff
@@ -776,6 +780,26 @@ dedicated `number` entities.
...
 
+{% details "Statistics" %}
+
+All statistical characteristics were removed from sensor attributes and are
+now selectable through `state_characteristic` as the state of the sensor
+directly.
+The parameter `state_characteristic` will become mandatory for configuration
+in a future release.
+
+Users will need to adapt their configuration and expose the desired characteristic.
```
Comment (frenck) [inline source/_posts/2021-12-11-release-202112.markdown]: Don't refer to the reader as "users". Use "you" instead.

File: source/_posts/2021-12-11-release-202112.markdown [hunk 3]
```diff
@@ -776,6 +780,26 @@ dedicated `number` entities.
...
+Additional template sensors to expose attributes are no longer necessary and
+existing ones will no longer work.
+
+Additionally, many more characteristics were added, including a few for a
+binary source sensor. Most characteristics were renamed. Please check
+documentation.
```
Comment (frenck) [inline source/_posts/2021-12-11-release-202112.markdown]:
```suggestion
Additionally, many more characteristics were added, including a few for a
binary source sensor. Most characteristics were renamed. Please check
the documentation.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_posts/2021-12-11-release-202112.markdown (hunk 2 above)
Comment (ThomDietrich) [reply source/_posts/2021-12-11-release-202112.markdown]: Done

File: source/_posts/2021-12-11-release-202112.markdown (hunk 1 above)
Comment (ThomDietrich) [reply source/_posts/2021-12-11-release-202112.markdown]: No hard feelings :) Even thought I thought this change is significant enough (in light of all the long term stats talk)

**Review by frenck — APPROVED** [review]
Thanks, @ThomDietrich 👍

**Pull request merged**